browse/templates/abs/404.html

```diff
@@ -39,10 +41,11 @@ <h1>Article {{ arxiv_id }} in the {{ archive_name }} archive not found</h1>
         archive, enter an article number: {{ archive_id }} / <input name="papernum" type="text" size="7"></p>
       </form>
 
-    {% elif reason=='deleted' and arxiv_id_latest %}
+    {% elif reason=='deleted' and message|string %}
       <h1>Article {{ arxiv_id_latest }} not found</h1>
-      <p><b>The article <i>{{ arxiv_id_latest }}</i> is recorded as having been
-      deleted from arXiv. The reason recorded is <i>"{{ message }}"</i>.</b></p>
+      <p><b>The article <i>{{ arxiv_id_latest }}</i> is recorded as having been deleted from arXiv.
```

Comment:
This needs to be "No record for {{ arxiv_id_latest }}", as per Slack discussion.

Reply:
Ones with no `reason` will be forced to `do_not_found` by line 3 and end up on line 29 and 30.